### ISSUE TITLE
Is HODome dataset available now？

### ISSUE THREAD

**Issue by wzyabcas** [Juzezhang/IKOL#1]

**Comment by Juzezhang** [comment]
Hi [wzyabcas](https://github.com/wzyabcas),
 
Thank you for your interest in this work！We will make our dataset available as soon as possible before CVPR conference. Feel free to contact us if you have any questions.

**Comment by Steve-Tod** [comment]
Hi,

Will HODome dataset be available any time soon? CVPR has passed for a few months.

**Comment by Juzezhang** [comment]
Dear @wzyabcas  and  @Steve-Tod,
 
Thank you for your interest in our work! We are currently in the process of preparing the NeuralDome dataset for release. Unfortunately, two main reason leads to the huge delay for opening. Firstly, certain failed cases within the dataset require manual removal and we need to reprocess the sequence to improve the quality of groundtruth. Secondly, our data is affected by privacy concerns related to facial recognition because some of the actors did not sign agreements at that time. Consequently, we are compelled to omit certain sequences.
 
Nevertheless, we can provide the test sets in advance. Please note that I'm currently in the process of uploading them to Google Cloud Drive (https://drive.google.com/drive/folders/1-QHvcwa71Wk7rdfnQrOyInqK-SWK6lRA?usp=sharing). Since the file size exceeds 2TB, it may take more than a few days to complete the upload. Free free to contact us if you have any questions!
 
Best regards, 
Juze Zhang

**Comment by xiexh20** [comment]
thanks Juze for releasing the test set. 
I am wondering will the training set also be released soon?

Best,
Xianghui

**Comment by Juzezhang** [comment]
Dear @xiexh20,

Sorry for the huge delay in this comment. The training set for Hodome has now been uploaded and is available for access on our website at https://juzezhang.github.io/NeuralDome/. Please find it within the website(https://juzezhang.github.io/NeuralDome/). And I will take the next few days to develop the neuraldome toolbox to facilitate its use within the community.

Best regards
Juze
